static/bidder-info/smrtconnect.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint: "https://amp.smrtconnect.com/openrtb2/auction?supply_id={{.SupplyId}}"
+endpointCompression: gzip
+maintainer:
+  email: "tech@smrtconnect.com"
```

Comment:
@psmrt We tried sending a verification mail to tech@smrtconnect.com but the mail bounced. Could you please check if this is the correct address and it is working

Reply:
Thank you @psmrt . We have sent an email to the updated address. Please reply to the mail with "Received" to complete the verification process.

Reply:
done